src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -2313,6 +2338,13 @@ bool QgsPostgresProvider::addFeatures( QgsFeatureList &flist, Flags flags )
         continue;
 
       QString fieldname = mAttributeFields.at( idx ).name();
+
+      if ( !mGeneratedValues.value( idx, QString() ).isEmpty() )
+      {
+        QgsDebugMsg( QStringLiteral( "Skipping field %1 (idx %2) which is GENERATED." ).arg( fieldname, idx ) );
```

Comment:
```suggestion
        QgsDebugMsg( QStringLiteral( "Skipping field %1 (idx %2) which is GENERATED." ).arg( fieldname, QString::number( idx ) ) );
```

This was calling the wrong overload. https://doc.qt.io/qt-5/qstring.html#arg

Reply:
Addressed by 5a56586